CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changes
 
+#### Features
```

Comment:
Please remove `Features` section and put your changelog under `Improvements` section

Reply:
Done.